command/street_merge.go

```diff
@@ -30,6 +30,7 @@ type config struct {
 	Format          string
 	Delim           string
 	ExtendedColumns bool
+	highwayTags     bool
```

Comment:
What is the purpose of this new `bool` field in the config struct?
I see it's being set `true/false` but doesn't seem to be read anywhere?

Reply:
Since it's lowercase (and therefore package scoped) I'm guessing it was something you were experimenting with but didn't end up using?